snippets/react-snippets.code-snippets

```diff
@@ -114,6 +114,21 @@
         ],
         "description": "Add `role='radio'` to a checkable interactive control. Use radio in place of checkbox if only one item in a group can be checked. Add `aria-checked` to indicate the state of the checkbox."
     },
+    "TableAria": {
+        "prefix": [
+            "TableAria",
+        ],
+        "body": [
+            "aria-colcount=''",
```

Comment:
I think you forgot role=table?

Reply:
except this all things are okay??

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role#associated_wai-aria_roles_states_and_properties

^ I have been adding the ones that mozilla specifies